src/components/form/input.tsx

```diff
@@ -258,7 +260,7 @@ const Input = ({
                         htmlFor={id}
                         label_color={label_color}
                     >
-                        {label}
+                        <Localize translate_text={label} />
```

Comment:
```suggestion
                      {label &&  <Localize translate_text={label} />}
```

Reply:
<img width="757" alt="Screenshot 2023-03-09 at 10 54 02 AM" src="https://user-images.githubusercontent.com/125247833/223904225-4996bbe2-11ba-4751-b479-b567b4f2d394.png">
this condition has been checked on top of it